frontend/src/state/migration/index.js

```diff
@@ -0,0 +1,31 @@
+const readAndDeleteFromLocalStorage = (key, defaultValue) => {
+  const val = localStorage[key];
```

Comment:
Should probably use localStorage.setItem

getItem and removeItem?

I guess that protects against accidentally mutating wrong things on the raw object?